SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -32,10 +32,10 @@ void USpatialPackageMapClient::Init(USpatialNetDriver* NetDriver, FTimerManager*
 	}
 }
 
-void GetSubobjects(UObject* Object, TArray<UObject*>& InSubobjects)
+void GetSubobjects(UObject* ParentObject, TArray<UObject*>& InSubobjects)
```

Comment:
Wait, none of these changes should be necessary. We always explicitly capture are variables, so they wouldn't be accessible within the lambda.

Reply:
Not a reason to block this PR obv - just curious what the deal is

Reply:
a different older compiler on Android that actually does fail on it. Switching to a newer version doesn't, but this is the Android compiler version officially supported by Unreal

Reply:
It's just strange, because for instance in this change, `Object` shouldn't be accessible internally. If you put `checkf(object)` does it compile?